HelloWorld/AppIdentity.juvix

```diff
@@ -0,0 +1,24 @@
+module AppIdentity;
+
+import Stdlib.Prelude open;
+import Applib open;
+
+--- The label for HelloWorld resources
+label : Nat := builtinAnomaEncode "HelloWorld";
+
+--- Compute the kind of the HelloWorld ;Resource;
+main (encodedLogic : Encoded Logic) : Kind :=
```

Comment:
since it is an introductory example, we could mention in a comment that only the logic and label are relevant for the kind.

Reply:
I've added a comment, thanks.